.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+/.github/CODEOWNERS @KingBain
+
+*       @ErikApption @Sean-Stilwell @davidreneuw @yjmrobert
+
+*.md    @deb-h576
+*.json  @deb-h576

```

Comment:
```suggestion
```
I would remove this, we don't have many json files other than the i18n (already caught by the next line) that are of comms importance.